packages/expo-image/src/Image.types.ts

```diff
@@ -303,6 +311,8 @@ export interface ImageNativeProps extends ImageProps {
   placeholder?: ImageSource[];
   contentPosition?: ImageContentPositionObject;
   transition?: ImageTransition | null;
+  animated?: boolean;
```

Comment:
I guess it should be `autoplay`, right?
```suggestion
  autoplay?: boolean;
```

Reply:
Oops 😅 Looks like just the weird issue I love to throw into something!